src/scene/composition/render-action.js

```diff
@@ -59,6 +62,12 @@ class RenderAction {
         this.viewBindGroups.length = 0;
     }
 
+    setupClears(camera, layer) {
+        this.clearColor = camera?.clearColorBuffer || layer.clearColorBuffer;
```

Comment:
I assume `||` is correct here instead of `??`?

Reply:
both work here equally.